pkg/proc/amd64_arch.go

```diff
@@ -356,13 +359,25 @@ var amd64DwarfToName = map[int]string{
 	66: "SW",
 }
 
+var amd64NameToDwarf = func() map[string]int {
```

Comment:
Clever. I went back and forth during the review on whether this should happen in an init function, but I kind of like the initialization happening where the var declaration is.

Reply:
I didn't come up with this, I saw it somewhere.